presto-main/src/main/java/io/prestosql/sql/planner/LogicalPlanner.java

```diff
@@ -549,21 +610,21 @@ private Expression noTruncationCast(Expression expression, Type fromType, Type t
         ResolvedFunction fail = metadata.resolveFunction(QualifiedName.of("fail"), fromTypes(VARCHAR));
 
         return new IfExpression(
-                // check if the trimmed value fits in the target type
-                new ComparisonExpression(
-                        GREATER_THAN_OR_EQUAL,
-                        new GenericLiteral("BIGINT", Integer.toString(targetLength)),
-                        new CoalesceExpression(
-                                new FunctionCall(
-                                        spaceTrimmedLength.toQualifiedName(),
-                                        ImmutableList.of(new Cast(expression, toSqlType(VARCHAR)))),
-                                new GenericLiteral("BIGINT", "0"))),
-                new Cast(expression, toSqlType(toType)),
-                new Cast(
-                        new FunctionCall(
-                                fail.toQualifiedName(),
-                                ImmutableList.of(new Cast(new StringLiteral("Cannot truncate non-space characters on INSERT"), toSqlType(VARCHAR)))),
-                        toSqlType(toType)));
+            // check if the trimmed value fits in the target type
+            new ComparisonExpression(
+                GREATER_THAN_OR_EQUAL,
+                new GenericLiteral("BIGINT", Integer.toString(targetLength)),
+                new CoalesceExpression(
+                    new FunctionCall(
+                        spaceTrimmedLength.toQualifiedName(),
+                        ImmutableList.of(new Cast(expression, toSqlType(VARCHAR)))),
+                    new GenericLiteral("BIGINT", "0"))),
+            new Cast(expression, toSqlType(toType)),
+            new Cast(
+                new FunctionCall(
+                    fail.toQualifiedName(),
+                    ImmutableList.of(new Cast(new StringLiteral("Cannot truncate non-space characters on INSERT"), toSqlType(VARCHAR)))),
+                toSqlType(toType)));
```

Comment:
Unrelated formatting change?

Reply:
Yes, sorry, I went back and reverted all such unintentional formatting changes in this file.